freqtrade/rpc/api_server/api_v1.py

```diff
@@ -135,13 +135,13 @@ def show_config(rpc: Optional[RPC] = Depends(get_rpc_optional), config=Depends(g
     return resp
 
 
-# /forcebuy is deprecated with short addition. use ForceEntry instead
-@router.post('/forceenter', response_model=ForceEnterResponse, tags=['trading'])
+# /forcebuy is deprecated with short addition. use Force_entry instead
+@router.post('/force_enter', response_model=ForceEnterResponse, tags=['trading'])
```

Comment:
for the API - please don't use this naming.
it's not used by people - and for an api, `/forceenter` is clearer.

Reply:
Changed. But https://stackoverflow.com/q/10302179/5033247